dev_guide/builds/build_inputs.adoc

```diff
@@ -257,6 +257,16 @@ xref:../../install_config/http_proxies.adoc#configuring-hosts-for-proxies-using-
 a global proxy for Git cloning using Ansible].
 endif::[]
 
+[NOTE]
+====
+For Pipeline strategy builds, given the current restrictions with the Git plug-in for Jenkins, any Git
+operations via the Git plug-in will not leverage the HTTP or HTTPS proxy defined in the `BuildConfig`.
+The Git plug-in only will use the the proxy configured in the Jenkins UI at the Plugin Manager panel.
+This proxy will then be used for all git interactions within Jenkins, across all jobs.
+Instruction on how to configure proxies via the Jenkins UI can be found
```

Comment:
s/Instruction/Instructions/

(but maybe that's a personal quirk?  i'm not sure Instruction is actually wrong.  we can let @bmcelvee decide)